src/main/java/TiTi/task/Deadline.java

```diff
@@ -0,0 +1,41 @@
+package TiTi.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents the Deadline task.
+ * Contain String description of the task.
+ * Contain String description of the date of task.
+ * Contain LocalDate object of the date of task (if applicable).
+ * Contain boolean value of whether the task has been completed.
```

Comment:
Should be `Contains` instead of `Contain`. These are probably overlooked but I noticed the same issue in several other places too.